### PR TITLE
Split cert-manager alerts per provider

### PR DESCRIPTION
**Pull request by T-Kukawka**
As agreed on KaaS sync we are splitting Cert Manager alerts per provider

This PR:

- adds/changes/removes etc

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Add Unit tests
- [x] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).
- [x] Alerting rules must have a comment documenting why it needs to exist.
- [x] Consider creating a dashboard (if it does not exist already) to help oncallers monitor the status of the issue.

### CODE REVIEW

**Review by QuentinBisson — APPROVED** [review]

**Review by QuantumEnigmaa — APPROVED** [review]

File: helm/prometheus-rules/Chart.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ engine: gotpl
 home: https://github.com/giantswarm/prometheus-rules
 icon: https://s.giantswarm.io/app-icons/1/png/default-app-light.png
 name: prometheus-rules
-appVersion: '[[ .AppVersion ]]'
-version: '[[ .Version ]]'
+appVersion: '2.65.0-b55df96be65b6b756f8efafed867dffbb2afc5bf'
+version: '2.65.0-b55df96be65b6b756f8efafed867dffbb2afc5bf'
```
Comment (calvix) [inline helm/prometheus-rules/Chart.yaml:8]: that does not seem right

Comment (T-Kukawka) [reply helm/prometheus-rules/Chart.yaml:8]: -,- make test changed that i guess

Comment (T-Kukawka) [reply helm/prometheus-rules/Chart.yaml]: fixed

File: helm/prometheus-rules/values.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@ name: prometheus-rules
 namespace: monitoring
 serviceType: managed
 project:
-  branch: '[[ .Branch ]]'
-  commit: '[[ .SHA ]]'
+  branch: 'cert-manager-split'
+  commit: 'b55df96be65b6b756f8efafed867dffbb2afc5bf'
```
Comment (njuettner) [inline helm/prometheus-rules/values.yaml:6]: Is that correct?

Comment (hervenicol) [reply helm/prometheus-rules/values.yaml:6]: nope, should not have been commited.

Comment (T-Kukawka) [reply helm/prometheus-rules/values.yaml]: my bad - fixed

**Pull request merged**